### PR TITLE
Updates for the database analysis guide

### PR DESCRIPTION
**Pull request by prees1**
A few improvements for the database analysis guide:

- Add new supported db version, Oracle 18c
- Add information about `analyze_workload` option

Doesn't impact rendered content:
- Remove redundant whitespace

### CODE REVIEW

File: pages/tidal tools/analyze_database.md [hunk 1]
```diff
@@ -95,6 +99,10 @@ databases:
 
 {% include note.html content="It is best to use quotations, either double or single, around the values in the configuration file. To avoid special characters, : { } [ ] , & * # ? | - < > = ! % @ \ \n from being interpreted. Single quotes are safest, if the value has a single quote within it, you can include it by using a two single quotations, ie. `'my''string'` - will become `my'string`." %}
 
+{% include tip.html content="Are you analyzing an Oracle database and need to
+only use features that are including the Oracle Standard Edition (SE) license?
```
Comment (pshevtsov) [inline pages/tidal tools/analyze_database.md:103]: “to only use features” or “to use only features”?

> features that are including the Oracle Standard Edition (SE) license

I think it should be “features … included in the … license”. WDYT?

Maybe let's rewrite this sentence into something like the following:

**Oracle Standard Edition (SE) user? Check out the advanced configuration section below to know how to perform your database analysis using only those features which are permitted by the terms of your Oracle license.**

WDYT?

File: pages/tidal tools/analyze_database.md [hunk 2]
```diff
@@ -133,3 +141,26 @@ You need to install Docker in order to complete the database analysis. This is b
 
 ##  What about security?
 The entire analysis takes place locally on your machine. The only data that is captured and sent from the analysis are the results of the analysis and metadata. No application data, source code, files or the contents of any files on your machine are ever copied or sent anywhere.
+
+## Advanced Configuration
+
+If you are analyzing an Oracle databaase and need to only use features that are
```
Comment (pshevtsov) [inline pages/tidal tools/analyze_database.md:147]: Let's rewrite it into something like the following:

**To comply with the terms of the Oracle Standard Edition (SE) license you should set the `analyze_workload` parameter to `false` in your database analysis configuration file. For example,**

WDYT?

File: pages/tidal tools/analyze_database.md [hunk 3]
```diff
@@ -133,3 +140,25 @@ You need to install Docker in order to complete the database analysis. This is b
...
+    engine: Oracle
+    host: 'my-db-host.com'
+    port: 1521
+    db_name: 'orcl'
+    user: 'tidal'
+    password: 'yoursecurepassword1234!'
+    name: 'My-Test-DB'
+```
+
+{% include note.html content="By setting this to `false` some results from the
```
Comment (pshevtsov) [inline pages/tidal tools/analyze_database.md:162]:
```suggestion
{% include note.html content="By setting this to `false`, some results from the
```

File: pages/tidal tools/analyze_database.md [hunk 4]
```diff
@@ -133,3 +140,25 @@ You need to install Docker in order to complete the database analysis. This is b
 
 ##  What about security?
 The entire analysis takes place locally on your machine. The only data that is captured and sent from the analysis are the results of the analysis and metadata. No application data, source code, files or the contents of any files on your machine are ever copied or sent anywhere.
+
+## Advanced Configuration
+
+To use Oracle features included only in the Oracle Standard Edition (SE) license, you
+can set the `analyze_workload` flag to `false` in your configuration file. For example:
```
Comment (pshevtsov) [inline pages/tidal tools/analyze_database.md:147]: It's not a **flag**, but **property** or **option**. WDYT?

Comment (prees1) [reply pages/tidal tools/analyze_database.md:147]: yep fair I'll switch to property.

**Review by pshevtsov — APPROVED** [review]
LGTM! :rocket:

**Pull request merged**